spring-cloud-stream-binder-kafka/pom.xml

```diff
@@ -27,6 +27,17 @@
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-stream</artifactId>
+			<version>1.2.2.BUILD-SNAPSHOT</version>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-core</artifactId>
+			<version>4.3.10.BUILD-SNAPSHOT</version>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-jmx</artifactId>
+			<version>4.3.10.BUILD-SNAPSHOT</version>
```

Comment:
I think this PR need some versions update as all others similar.
I mean that we don't need this `4.3.10.BUILD-SNAPSHOT`, the GA is polled by SCSt transitively.
Seems for me the same is for SIK